### ISSUE TITLE
Only ship required mfc dlls

### ISSUE THREAD

**Issue by mhammond** [mhammond/pywin32#1869]
The binary distributions ship:

```
5,793,048 mfc140.dll
5,821,208 mfc140u.dll
  102,680 mfcm140.dll
  102,680 mfcm140u.dll
```

and I'm fairly sure only `mfc140u.dll` is needed. That would reduce the size of the distros significantly.

[This code](https://github.com/mhammond/pywin32/blob/256648cb84734d79094d9c9cac569c59705b22c1/setup.py#L770-L773) only taking the "canary") seems like what we want, but will need testing on a "clean" install.

**Comment by alex-eri** [comment]
maybe ship it separatly like wine?

**Comment by Avasam** [comment]
Looks like this has been fixed in 306. Only 1 DLL in win32 wheels instead of 4:
![image](https://github.com/user-attachments/assets/7ec1acea-4c2a-4bdc-8bd6-ac8654127755)
Likely fixed by #1941 if I were to guess (@kxrob 's description mentions this too)
